cmd/config.go

```diff
@@ -161,6 +161,15 @@ type Config struct {
 
 	PA PAConfig
 
+	Statistics struct {
```

Comment:
Update: the config files are now under test/config/*.json, and each main.go defines its own config struct.